bench/suite/src/main/scala/org/scalameta/bench/SymbolParser.scala

```diff
@@ -0,0 +1,29 @@
+package org.scalameta.bench
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations.{Benchmark, BenchmarkMode, Measurement, Mode, Fork}
+import org.langmeta.semanticdb.Symbol
+import org.langmeta.semanticdb.Signature
+
+object SymbolParser {
+  val symbol: String = Symbol.Global(
+    Symbol.Global(
+      Symbol.Global(
+        Symbol.Global(
+          Symbol.None,
+          Signature.Term("_root_")),
+        Signature.Term("scala")),
+      Signature.Term("Predef")),
+    Signature.Type("Int")).syntax
+}
+class SymbolParser {
+
+  @Benchmark
+  @BenchmarkMode(Array(Mode.Throughput))
+  @Fork(value = 1)
+  @Measurement(iterations = 5, time = 1, timeUnit = TimeUnit.SECONDS)
+  def run: Unit = {
```

Comment:
The return type should be `Symbol`. [Otherwise it is dead code and might be optimised away](https://github.com/ktoso/sbt-jmh/blob/a6f5b8e8e077f77c52aab3f5c7c4a18581bb8ece/plugin/src/sbt-test/sbt-jmh/run/src/main/scala/org/openjdk/jmh/samples/JMHSample_08_DeadCode.scala#L52-L54).